docs/quickstart.rst

```diff
@@ -77,3 +76,42 @@ On other systems:
     $ python3 example_bot.py
 
 Now you can try playing around with your basic bot.
+
+A Minimal Bot with Slash Commands
+---------------
+
+As a continuation, let's create a bot that registers a simple slash command!
+
+It looks something like this:
+
+.. code-block:: python3
+
+    import discord
+
+    bot = discord.Bot()
+
+    @bot.event
+    async def on_ready():
+        print(f"We have logged in as {bot.user}")
+
+    @bot.slash_command(name="hello", guild_ids=[your, guild_ids, here])
+    async def hello_slash(ctx):
+        await ctx.respond("Hello!")
+
+    bot.run("your token here")
+
+Let's look at the differences compared to the previous example, step-by-step:
+
+1. The first line remains unchanged.
+2. Next, we create an instance of :class:`Bot`. This is different from :class:`Client`, as it supports
+   slash command creation and other features, while inheriting all the features of :class:`Client`.
+3. We then use the :meth:`Bot.slash_command` decorator to register a new slash command. The first
+   keyword-argument, ``name``, sets the command's name to ``hello``. The ``guild_ids`` attribute contains
```

Comment:
This makes name sound like a required argument

Reply:
I think I'll actually just remove name entirely to simplify this example.